cornice/tests/test_cors.py

```diff
@@ -228,15 +228,6 @@ def test_resp_dont_include_allow_origin(self):
         self.assertNotIn('Access-Control-Allow-Origin', resp.headers)
         self.assertEqual(resp.json, 'squirels')
 
-    def test_resp_allow_origin_wildcard(self):
-        resp = self.app.options(
-            '/cors_klass',
-            status=200,
-            headers={
-                'Origin': 'lolnet.org',
-                'Access-Control-Request-Method': 'POST'})
-        self.assertEqual(resp.headers['Access-Control-Allow-Origin'], '*')
```

Comment:
Yes! I believe this fix then misses a test that exercise exactly what you described (we should check that the header is returned correctly, but not sent).